aep/0126.yaml

```diff
@@ -0,0 +1,71 @@
+functionsDir: ../functions
+functions:
+  - aep-126-enum-case-consistent
+  - aep-126-enum-null-first
+  - aep-126-enum-nullable-declaration
+  - aep-126-no-standard-value-enums
+
+rules:
+  aep-126-enum-type-string:
+    description: Enumerated fields should use type string, not integer or other types.
+    message: Enum field "{{property}}" should have type "string", not "{{error}}".
+    severity: error
+    formats: ['oas2', 'oas3']
+    given: $..[?(@property === 'enum')]^
+    then:
+      field: type
+      function: schema
+      functionOptions:
+        schema:
+          oneOf:
+            - const: string
+            - type: array
+              contains:
+                const: string
+              maxItems: 2
```

Comment:
I guess `maxItems` is here to allow "null" but it also allows other things that we'd want to flag. E.g. 'type: ["string", "number"]` should fail but I think would not currently.